src/core.jl

```diff
@@ -92,6 +92,25 @@ function render(mode::RenderMode, actual)
     println("-------------------------------")
 end
 
+
+default_rendermode(::TestMode, ::File, actual) = BeforeAfterFull()
+
+# plain TXT
+default_rendermode(::TestMode, ::File{format"TXT"}, actual) = BeforeAfterLimited()
+default_rendermode(::TestMode, ::File{format"TXT"}, actual::AbstractString) = Diff()
+default_rendermode(::TestMode, ::File{format"TXT"}, actual::Number) = BeforeAfterFull()
+
+# SHA256
+default_rendermode(::TestMode, ::File{format"SHA256"}, actual) = BeforeAfterFull()
```

Comment:
This is redundant; I put it here to make sure `SHA256` is not influenced by potential future changes of the `::File` fallback.